types/sector_class.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+// SectorClass represents the miner's chosen sector size and PoSt/seal proof
+// partitions.
+type SectorClass struct {
+	PostProofPartitions PoStProofPartitions
+	SealProofPartitions SealProofPartitions
+	SectorSize          SectorSize
```

Comment:
Question from curiosity: do you imagine a PoStClass to capture things like number of sectors aggregated under a post?  Is that stuff totally separate from sector class?

Reply:
@ZenGround0 there's no direct relationship between the number of sectors which are challenged during PoSt generation and the number of PoSt proof partitions. 

So, 

> Is that stuff totally separate from sector class?

Yep, that stuff is separate from sector class.